### PR TITLE
fix(utils): manally convert from b64 to b64url to avoid web polyfill i…

### PR DESCRIPTION
**Pull request by dtfiedler**
…ssues

Somewhat related to polyfill issues not handling `base64url` correctly - https://github.com/feross/buffer/pull/314/files

### CODE REVIEW

File: src/utils/base64.ts [hunk 1]
```diff
@@ -16,12 +16,39 @@
  */
 import { createHash } from 'crypto';
 
-export function fromB64Url(input: string): Buffer {
-  return Buffer.from(input, 'base64');
+// safely encodes and decodes base64url strings to and from buffers
```
Comment (fedellen) [inline src/utils/base64.ts:19]: is this sourced from somewhere?

File: src/utils/base64.ts [hunk 2]
```diff
@@ -16,12 +16,39 @@
...
+}
+
+function base64urlFromBase64(str: string) {
+  return str
+    .replaceAll(BASE64_CHAR_62, BASE64URL_CHAR_62)
+    .replaceAll(BASE64_CHAR_63, BASE64URL_CHAR_63)
+    .replaceAll(BASE64_PADDING, '');
+}
+
+export function fromB64Url(str: string): Buffer {
```
Comment (fedellen) [inline src/utils/base64.ts:44]: nit: naming inconsistency from `base64` to `B64`

**Review by atticusofsparta — APPROVED** [review]

File: src/utils/base64.ts (hunk 1 above)
Comment (fedellen) [reply src/utils/base64.ts:19]: ah, from desc -- https://github.com/feross/buffer/pull/314/files

**Review by fedellen — DISMISSED** [review]

**Review by fedellen — APPROVED** [review]

**Pull request merged**